files/bootstrap.sh

```diff
@@ -392,6 +399,22 @@ Environment='KUBELET_EXTRA_ARGS=$KUBELET_EXTRA_ARGS'
 EOF
 fi
 
+# Replace dockerd with containerd
+if [[ "$CONTAINER_RUNTIME" -eq "containerd" ]]; then
```

Comment:
@rothgar I believe this is incorrect because `-eq` operator is used for numeric comparison, not string comparison. [1] This should be`=`.

[1] https://tldp.org/LDP/abs/html/comparison-ops.html#ICOMPARISON1